osu.Game/Rulesets/Mods/UnknownMod.cs

```diff
@@ -16,6 +16,7 @@ public class UnknownMod : Mod
         public override double ScoreMultiplier => 0;
 
         public override bool UserPlayable => false;
+        public override bool PlayableInMultiplayer => true;
```

Comment:
This override made the naming `PlayableInMultiplayer` fall apart. Thinking about using `SelectableInMultiplayer` or just going the simple way and use `AllowInMultiplayer`.

Reply:
Nevermind, this override was just wrong.